pkg/systemd/generate/pods.go

```diff
@@ -364,8 +364,10 @@ func executePodTemplate(info *podInfo, options entities.GenerateSystemdOptions)
 			podCreateArgs = append(podCreateArgs, "--replace")
 		}
 
+		if !hasPodExitPolicy(append(startCommand, podCreateArgs...)) {
+			startCommand = append(startCommand, "--exit-policy=stop")
```

Comment:
@dcermak here.  Wouldn't it have been sufficient to just move the old function call here?  My brain is in an afternoon shape, so I may be missing something.

Reply:
Yes, that would be possible, but I found the resulting code harder to read as it requires setting a flag whether we have to add the `exit-policy`. I found it simpler to read like this.

Reply:
Thanks! I am OK with the change but usually do not like bug fixes and refactorings in the same commit.